### PR TITLE
[skip e2e] Add script for building deb package

### PR DESCRIPTION
**Pull request by Bennu-Li**
Signed-off-by: Bennu-Li <yunmei.li@zilliz.com>

### CODE REVIEW

**Review by LoveEachDay — APPROVED** [review]
/lgtm

File: build/deb/debian/copyright [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: milvus
+Source: https://github.com/milvus-io/milvus
+
+Files: *
+Copyright: 2019-2021, Zilliz <yunmei,li@zilliz.com>
```
Comment (yanliang567) [inline build/deb/debian/copyright:6]: is it okay to leave a personal mail in copyright? and shall we update the copyright to 2022

**Review by LoveEachDay — APPROVED** [review]
/lgtm

**Pull request merged**